app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2009-2014 Martin Szulecki
+	2008-2010 Jonathan Beck
+	2009 Nikias Bassen
+	2008 Matt Colyer"
+LICENSE="GNU GPL v2
+	GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.libimobiledevice.org/downloads/$portName-$portVersion.tar.bz2"
```

Comment:
Oops, I had noticed a small error here but had forgotten to report it. We can't use $portName here because it's not always equal to "libplist". So we need to have:
```
SOURCE_URI="http://www.libimobiledevice.org/downloads/libplist-$portVersion.tar.bz2" 
```
BTW, when you fix this, maybe it's a good idea, after all, to change SECONDARY_ARCHITECTURES to:
```
SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
```
This way you won't need to make another update in case we find out that the gcc2 patch is not desirable.